### PR TITLE
Add CI

### PR DESCRIPTION
**Pull request by nikeee**

### CODE REVIEW

File: src/D2DWinForm/D2DWinForm.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
   <!-- Build properties -->
   <PropertyGroup>
-    <TargetFrameworks>net462;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net462;net6.0-windows</TargetFrameworks>
```
Comment (drewnoakes) [inline src/D2DWinForm/D2DWinForm.csproj:7]: .NET 6 is nearly out of support. We could move these to .NET 8.

File: src/d2dlib/D2DLib.vcxproj [hunk 2]
```diff
@@ -97,7 +97,7 @@
   </PropertyGroup>
   <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <ClCompile>
-      <PrecompiledHeader>Use</PrecompiledHeader>
+      <PrecompiledHeader>NotUsing</PrecompiledHeader>
```
Comment (drewnoakes) [inline src/d2dlib/D2DLib.vcxproj:100]: Why remove precompiled headers? I don't see an explanation for that change.

**Review by drewnoakes — APPROVED** [review]
Looks good to me.

File: src/D2DWinForm/D2DWinForm.csproj (hunk 1 above)
Comment (nikeee) [reply src/D2DWinForm/D2DWinForm.csproj:7]: Resolved in e25d1c2337bcf07db62

File: src/d2dlib/D2DLib.vcxproj (hunk 2 above)
Comment (nikeee) [reply src/d2dlib/D2DLib.vcxproj:100]: Reverted in 48fa7090323981

**Comment by nikeee** [comment]
Addressed some review issues. Better late than never :)

**Comment by jingwood** [comment]
Thanks @nikeee! I will also check why it fails, and get merge this once resolved.

**Comment by jingwood** [comment]
I'm going to fix it after merging.

**Comment by jingwood** [comment]
If a dotnet project inlcudes a vc++ project, current `dotnet build` will fails. It seems a known issue and no plans to get it fixed.
https://github.com/dotnet/sdk/issues/10239

This causes the error.
https://github.com/jingwood/d2dlib/blob/master/src/D2DLibExport/D2DLibExport.csproj#L49

So the current CI will be temporarily disabled until get a solution.

**Pull request merged**